controllers/cluster.weave.works/existinginframachine_controller.go

```diff
@@ -724,12 +694,15 @@ func (a *ExistingInfraMachineReconciler) performActualUpdate(
 	}); err != nil {
 		return err
 	}
+	log.Info("repaving node...")
```

Comment:
This message needs additional context to be meaningful in the logs.  Same message for the next 2 log messages.

Reply:
Yes and no. All those messages are basically "trace" messages intended merely to be used to understand what step you're on. There's no deep meaning to the messages other than, "I'm here". I find them super helpful when debugging and they would be no more helpful with more info; they would just take up more space.